src/views/emails/template-parts/body/order/payment-info.php

```diff
@@ -27,15 +28,21 @@
 	return;
 }
 
+// Make sure payment gateway name is uppercase. If gateway is PayPal, ensure proper capitalization.
+$gateway_name = ucwords( $order->gateway );
+if ( 'Paypal' === $gateway_name ) {
+	$gateway_name = esc_html__( 'PayPal', 'event-tickets' );
+}
+
```

Comment:
There is a `get_label` method on gateways. Using that should be easier: https://github.com/the-events-calendar/event-tickets/blob/1a857b5135c22927fa9b3b5f187d7aef95e97028/src/Tickets/Commerce/Gateways/PayPal/Gateway.php#L73

Reply:
Yep. I was actually working on that. Got a new solution for it, now.